docs/migration.md

```diff
@@ -14,22 +14,22 @@ Before doing any migration **back up your database** in case the whole process g
 
 We introduced a new environment variable, `WEBSITE_NAME`. It identifies your _tf2pickup.org_ instance uniquely; for now, it will be used by the new [logs.tf](https://logs.tf/) uploader, but more use-cases are surely coming.
 
-```
-WEBSITE_NAME=tf2pickup.pl
+```env
+WEBSITE_NAME=tf2pickup.fi
 ```
 
 We also added support for expansion of environment variables, so now you can re-use your `WEBSITE_NAME`, for example:
 
-```
+```env
 BOT_NAME=${WEBSITE_NAME}
 ```
 
 ### Redis
 
 The new version requires a [Redis](https://redis.io/) database; it is used to cache some data and store game logs. Follow [site components deployment](site-components-deployment#docker-composeyml-for-the-website-only) documentation to learn how to set it up.
 
-```
-REDIS_URL=redis://localhost:6379
+```env
+REDIS_URL=redis://tf2pickup-fi-redis:6379
```

Comment:
Leave `localhost` as redis host here. In terms of migration, we're not in scope of the tf2pickup.fi deploy case, so it might be misleading.

Reply:
Leaving `localhost` would be more misleading in my opinion, because that means that the backend should connect to itself to get access to Redis. I put a different name delibrately to show off that you are supposed to set some name which represents Redis container's name or hostname.